### PR TITLE
Update auth service and frontend and add nos service

### PR DESCRIPTION
**Pull request by Cito**

### CODE REVIEW

File: charts/notification-orchestration/values.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+config_prefix: ns
+
+parameters:
+  default:
+    access_request_allowed_event_type: access_request_allowed
+    access_request_created_event_type: access_request_created
+    access_request_denied_event_type: access_request_denied
+    access_request_events_topic: access_requests
+    central_data_stewardship_email: ""
+    db_name: users
```
Comment (ckaipf) [inline charts/notification-orchestration/values.yaml:27]:
```suggestion
    db_name: nos
```
See DM in slack

File: charts/notification-orchestration/values.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+args: []
+command: []
+horizontalPodAutoscaler:
+  enabled: false
+  averageCpuUtilization: 75
+  averageMemoryUtilization: 75
+image:
+  name: "ghga/notification-service"
```
Comment (ckaipf) [inline charts/notification-orchestration/values.yaml:8]: Correct?

File: charts/notification-orchestration/values.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+podAnnotations: {}
+resources:
+  limits:
+    cpu: "500m"
+    memory: "256M"
+  requests:
+    cpu: "100m"
+    memory: "64M"
+
+config_prefix: ns
```
Comment (ckaipf) [inline charts/notification-orchestration/values.yaml:18]:
```suggestion
config_prefix: nos
```

Comment (ckaipf) [reply charts/notification-orchestration/values.yaml:18]: - [ ] Add also to config in helmfile, it is missing there

File: charts/notification-orchestration/values.yaml (hunk 2 above)
Comment (Cito) [reply charts/notification-orchestration/values.yaml]: No, fixed it.

File: charts/notification-orchestration/values.yaml (hunk 3 above)
Comment (Cito) [reply charts/notification-orchestration/values.yaml]: Fixed it already.

**Review by ckaipf — APPROVED** [review]

**Review by dontseyit — APPROVED** [review]

**Pull request merged**